Form1.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+
+
+namespace Conversion_Calculator
+{
+     
+    
+    public partial class Form1 : Form
+    {
+
+        double userInput = 0;
+        double _finalAnswer = 0;
+        double GALLON_TOTAL_PER_POUND = 8.33;
+        double GRAMS_TOTAL_PER_POUND = 453.592;
+        double GRAMS_TOTAL_PER_OUNCE = 28.349;
+        double KG_TOTAL_PER_POUND = 0.453;
+        double LITERS_TOTAL_PER_GAL = 0.264172;
+        double GAL_TOTAL_PER_LITER = 3.78541;
+
+
+
+
+        public Form1()
+        {
+            InitializeComponent();
+            tbUserInput.Select();
+           
+        }
+
+        
+
+        private void textBox1_TextChanged(object sender, EventArgs e)
+        {
+            string.IsNullOrWhiteSpace(tbUserInput.Text);// Is this the problem?

```

Comment:
this returns a bool, true if it's null or empty. This needs to be in an if statement